src/v2/Apps/Partner/partnerRoutes.tsx

```diff
@@ -91,10 +90,11 @@ export const partnerRoutes: RouteConfig[] = [
       },
       {
         getComponent: () => ArtistsRoute,
-        path: "artists",
+        path: "artist(s|/):artistId?",
```

Comment:
Added handling of both `/artists` and `/artist/:artistId` routes. but it's not an ideal variant. it also handles `/artist/` route. Maybe it would be better split it into two routes on the same level, wrap a `PartnerArtistDetailsList` component in `QueryRenderer`, and put it on both pages. What do you think?

Reply:
Just to clarify, it's not a question about child routes. I just think that it may be better to split the current route `artist(s|/):artistId?` into two new:
`/artists`
`/artist/:artistId`

Reply:
Oh yeah this looks a little odd 😅.

If the `/artist/:artistId` route is only jumping you to the correct part of the list, then maybe we should just keep it as `/artists/:artistId` which would remove this weirdness.

Curious if @damassi has thoughts on this or on your original question on how to organize these child routes.

Reply:
@sweir27  Updated current routing to using `/artists/:artistId?`. I think it's okay for now. I also can add support of nesting routes if it makes sense, but it would be better to make a separate PR for that.